TTYC.Constants/ConfigurationConstants.cs

```diff
@@ -2,6 +2,13 @@
 {
 	public static class ConfigurationConstants
 	{
-		public const string DefaultConnection = "DefaultConnection";
+		public struct SecurityOptions
+		{
+			public const string DefaultConnection = "DefaultConnection";
```

Comment:
I think you confused that... Security options looks more like InterCount, SubkeyLength and SaltSize.
And I think PasswordHashParams is a better name for the section